### ISSUE TITLE
Auto deployment of peril

### ISSUE THREAD

**Issue by pepibumur** [tuist/peril-settings#2]
Changes in peril-settings dependencies, [like this one](https://github.com/tuist/peril-settings/pull/1), require deploying Peril to Heroku manually.

We could automate this by doing the deployment on CI for every commit in master

**Comment by Austinate** [comment]
Steps to configure:
- Enable [GitHub integration](
https://devcenter.heroku.com/articles/github-integration#enabling-github-integration) at [Heroku Dashboard](https://dashboard.heroku.com/) (You'll need to approve Heroku usage inside GitHub org if it's not done yet)
- Enable automatic deploys as described[here](https://devcenter.heroku.com/articles/github-integration#automatic-deploys).

I'd like to help more but there is obviously no way without access to repo settings :)

**Comment by pepibumur** [comment]
I don't think that would work because we are not deploying this repository but Peril's. We'd need to set up a CI pipeline that pushes the latest commits from the Peril repository to the Heroku repository.

[Here](https://github.com/danger/peril/blob/master/docs/updating_peril.md)'s described what we'd need to do on CI.
